gen/MotController_data.yml

```diff
@@ -95,6 +95,9 @@ c_MotController_Create1:
       'limit_switch_source_rev': LimitSwitchSource.Deactivated,
       'limit_switch_follow_for': deviceId,
       'limit_switch_follow_rev': deviceId,
+
+      'clear_pos_on_limit_fwd': 0,
```

Comment:
Better to store whether these are set using a bool.

Reply:
configGetParameter and configSetParameter are autogenned to require floats, which makes that awkward

Reply:
You should probably fix the tests to actually test this then.